frontend/src/components/Contexts/AssetContext.tsx

```diff
@@ -10,16 +10,18 @@ interface Props {
 }
 
 const defaultAsset = {
-    asset: 'test',
+    asset: '',
     switchAsset: (newAsset: string) => {},
 }
 
 export const AssetContext = createContext<IAssetContext>(defaultAsset)
 
 export const AssetProvider: FC<Props> = ({ children }) => {
-    const [asset, setAsset] = useState(defaultAsset.asset)
+    const prevAsset = sessionStorage.getItem('assetString')
```

Comment:
We could consider using localStorage instead of session here, that way we store the last Asset for the entire browser, even if its closed. I imagine this is more common for users of flotilla, as one use/browser/computer will most likely only be used for one plant

Reply:
Agree